src/pages/MonthlyGood/MonthlyGoodSetupPageControl.vue

```diff
@@ -220,17 +220,25 @@
 								</p>
 							</div>
 
-							<div v-if="!hasBillingAgreement">
-								<div class="paypal-wrapper">
-									<div class="paypal-cover" v-if="$v.$invalid"></div>
-									<pay-pal-mg
-										:amount="totalCombinedDeposit"
-										@complete-transaction="submitMonthlyGood()"
-									/>
-									<p class="small-text">
-										Thanks to PayPal, Kiva receives free payment processing.
-									</p>
-								</div>
+							<div class="payment-dropin-wrapper" v-if="showDropInPayments">
```

Comment:
This will show if exp for bt drop in is active. I am assuming that there is no change in the drop in UI if there is a billing agreement or not

Reply:
That's correct we'll show the dropin ui if they have an active billing agreement which will allow a choice of vaulted payment methods for this subscription.